### ISSUE TITLE
Add an "Is Slot Grabbable" LogiX node

### ISSUE THREAD

**Issue by epicEaston197** [Neos-Metaverse/NeosPublic#3820]
### Is your tweak request related to a problem? Please describe.

the is grabbable grabbable node would basically check the slot for the grabbable component and see if the grabbable component is enabled or not

### Describe what would you like tweaked

a Logix node that takes a slot input to see if an object is grabbable or not

### Describe alternatives you've considered

maybe I'm doing something wrong but I can't tell if an item is grabbable or not so I can't stop it from being parented to my grabbing system

### Additional context

so basically I was creating an item that would be able to grab items and such but I can't see if the world is an item or not resulting in the world being grabbable according to the system 
if there is a way to avoid this without this tweak please tell me

**Comment by Zyzyl** [comment]
AFAIK there isn't an "Is Grabbable Grabbable" LogiX node. Did you mean "Is Grabbable Grabbed" or "Is Grabbable Scalable"? Or are you asking for a new node to be implemented?

From what you've said, I understood that you'd like to check whether a Slot has a Grabbable component and whether that component is enabled. If that's correct, I suspect this would be implicitly solved by component access. Otherwise this issue seems relevant https://github.com/Neos-Metaverse/NeosPublic/issues/424

**Comment by epicEaston197** [comment]
> AFAIK there isn't an "Is Grabbable Grabbable" LogiX node. Did you mean "Is Grabbable Grabbed" or "Is Grabbable Scalable"? Or are you asking for a new node to be implemented?
> 
> From what you've said, I understood that you'd like to check whether a Slot has a Grabbable component and whether that component is enabled. If that's correct, I suspect this would be implicitly solved by component access. Otherwise this issue seems relevant #424

oh sorry I misunderstood the input 
thank you I've updated the issue
somebody with permissions can you make this a feature request instead of a tweak?

I do not believe component accesses necessary that's a bit Overkill I just want a node that takes a slot input to tell me if that object is grabbable or not

**Comment by Zyzyl** [comment]
Sure, I've changed the tags. I've also updated the name of the requested node to be closer to the desired function.

**Comment by JackTheFoxOtter** [comment]
I think this does fall into the component access category. There are quite a few situations in which you only have the slot and want to know if a specific component exists on it, like getting the image texture from a picture or the text from a text field. There could be nodes implemented for all of these use-cases, but I do believe a more general component-level access is probably the better solution as that could solve all of those issues in a standard way.

You could run into issues when the same component exists multiple times on the same slot. Granted this is unlikely to happen with grabbable, but it's still possible.

For your specific use-case though. have you tried using the default grabbing mechanics Neos already offers for your system? You might be able to find a cleaner solution with the default tools, feel free to hit me up in-game and we can see if we figure an alternative solution out.

**Comment by epicEaston197** [comment]
> I think this does fall into the component access category. There are quite a few situations in which you only have the slot and want to know if a specific component exists on it, like getting the image texture from a picture or the text from a text field. There could be nodes implemented for all of these use-cases, but I do believe a more general component-level access is probably the better solution as that could solve all of those issues in a standard way.
> 
> You could run into issues when the same component exists multiple times on the same slot. Granted this is unlikely to happen with grabbable, but it's still possible.
> 
> For your specific use-case though. have you tried using the default grabbing mechanics Neos already offers for your system? You might be able to find a cleaner solution with the default tools, feel free to hit me up in-game and we can see if we figure an alternative solution out.

what do you mean default grabbing tools I don't know how that's even possible with how my system is set up
